### PR TITLE
Add GitHub Actions

### PR DESCRIPTION
**Pull request by RodrigoPereyraDiaz**
Add _Github Actions_ for
- continuous-integration -> _.NET_ build and test
- continues delivery of docker images
-- To _Github Packages_
-- To _DockerHub_
- code format verification
- conventional commits verification

### CODE REVIEW

File: .github/workflows/docker-continuous-delivery.yml [hunk 1]
```diff
@@ -0,0 +1,90 @@
+name: Continuous Delivery
+
+on:
+  push:
+    branches: ["main", "INT"]
```
Comment (hrocca) [inline .github/workflows/docker-continuous-delivery.yml:5]: shouldn't we also include the QA branch?

**Comment by hrocca** [comment]
maybe, in another scope of this task, in [code-format-verification.yml](https://github.com/FromDoppler/conversion-tool/pull/166/files#diff-736377426120557665901a701acb91a58cd5da606d15e33e9e67b4f1b314b9a1) file, we can think about reviewing the performance of the format verification tasks.
I think we could use a single image and the different actual jobs that are multiple steps.
Good job!!!
:rocket:

File: .github/workflows/docker-continuous-delivery.yml (hunk 1 above)
Comment (RodrigoPereyraDiaz) [reply .github/workflows/docker-continuous-delivery.yml:5]: In general, we use the `main` branch in the _QA_ environment before generating a semantic version release for _Production_

File: .github/workflows/docker-continuous-delivery.yml [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    tags:
+      - v*.*.*
+
+env:
+  DOCKER_FILE: ConversionTool/Dockerfile
```
Comment (RodrigoPereyraDiaz) [inline .github/workflows/docker-continuous-delivery.yml:10]: Could be nice fine a way to generalize this variable

File: .github/workflows/docker-continuous-delivery.yml [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+          tags: ${{ steps.meta.outputs.tags }}
+          labels: ${{ steps.meta.outputs.labels }}
+          build-args: version=${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:${{ env.SOURCE_VERSION }}
+
+  docker-hub:
+    name: Docker image to Docker Hub
+    runs-on: ubuntu-latest
+    env:
+      REGISTRY: docker.io
+      IMAGE_NAME: ${{ secrets.DOCKERHUB_USERNAME }}/conversion-tool
```
Comment (RodrigoPereyraDiaz) [inline .github/workflows/docker-continuous-delivery.yml:61]: Here use the explicit name for image, could be nice generalize in some way

**Pull request merged**